petriflow.schema.xsd

```diff
@@ -603,9 +603,7 @@
     </xs:simpleContent>
   </xs:complexType>
   <xs:simpleType name="remote">
-    <xs:restriction base="xs:string">
-      <xs:maxLength value="0"/>
-    </xs:restriction>
+    <xs:restriction base="xs:string"/>

```

Comment:
If there is no restriction please remove this simple type and change the element `remote`  in complex type `data` to type `xs:string`